README.md

```diff
@@ -43,18 +43,16 @@ guide the learner’s interaction with the component.
 
 **\_graphic** (object): The image that constitutes the component. It contains values for **alt**, **large**, and **small**.
 
->**alt** (string): This text becomes the image’s `alt` attribute. 
+>**alt** (string): The alternative text for this image. Assign [alt text](https://github.com/adaptlearning/adapt_framework/wiki/Providing-good-alt-text) to images that convey course content only.
```

Comment:
@moloko, should this be updated as per your schema suggestion? https://github.com/adaptlearning/adapt-contrib-graphic/blob/9dc9eb765cbc630fbc2cb7c735bf0edac39376ac/properties.schema#L45

Reply:
yes, why not (I focussed more on the schema help text since it didn't include the link)

Reply:
I'll leave as is. The link gives context as you've pointed out.